src/components/Garden/ModalFlows/CreateDecisionScreens/CreateDecisionScreens.js

```diff
@@ -0,0 +1,76 @@
+import React, { useCallback, useEffect, useMemo, useState } from 'react'
+import ActionFees from './ActionFees'
+import CreateDecisionRequirements from './CreateDecisionRequirements'
+import ModalFlowBase from '../ModalFlowBase'
+
+import { useAgreement } from '@hooks/useAgreement'
+import { useStakingState } from '@providers/Staking'
+import { useWallet } from '@providers/Wallet'
+
+function CreateDecisionScreens({ onComplete, onCreateTransaction }) {
+  const [loading, setLoading] = useState(true)
+  const [transactions, setTransactions] = useState([])
+  const { account } = useWallet()
+  const [agreement, agreementLoading] = useAgreement()
+  const { stakeManagement, loading: stakingLoading } = useStakingState()
+
+  useEffect(() => {
+    setLoading(true)
+    setTransactions([])
+  }, [account])
+
+  useEffect(() => {
+    setLoading(agreementLoading || stakingLoading)
+  }, [agreementLoading, stakingLoading])
+
+  const getTransactions = useCallback(
+    async onComplete => {
+      const intent = await onCreateTransaction()
+      setTransactions(intent)
+      onComplete()
+    },
+    [onCreateTransaction]
+  )
+
+  const screens = useMemo(
+    () =>
+      stakingLoading
```

Comment:
@rperez89 I was doing a review of the whole EVM executor. Shouldn't be the `loading` variable here instead of the `stakingLoading`?